meta/sai_meta.cpp

```diff
@@ -6024,6 +6063,35 @@ void meta_sai_on_fdb_event_single(
 
     std::string key_fdb = sai_serialize_object_meta_key(meta_key_fdb);
 
+    /*
+     * Because we could receive fdb event's before orch agent will query or
+     * create bridge/vlan/bridge port we should snoop here new OIDs and put
+     * them in local DB.
+     *
+     * Unfortunately we don't have a way to check whether those OIDs are correct
+     * or whether there maybe some bug in vendor SAI and for example is sending
+     * invalid OIDs in those event's. Also sai_object_type_query can return
+     * valid object type for OID, but this does not guarantee that this OID is
+     * valid, for example one of existing bridge ports that orch agent didn't
+     * query yet.
+     */
```

Comment:
this PR add visibility to this issue, in such case you can print out warning message. but it does not solve the issue.

Reply:
yes, that was intention of this PR